pymc3/step_methods/elliptical_slice.py

```diff
@@ -0,0 +1,67 @@
+# Elliptical slice sampler
+
+import numpy as np
+import numpy.random as nr
+import theano.tensor as tt
+
+from .arraystep import ArrayStep, Competence
+from ..model import modelcontext
+from ..theanof import inputvars
+from ..vartypes import continuous_types
+from ..distributions import draw_values
+
+__all__ = ['EllipticalSlice']
+
+
+class EllipticalSlice(ArrayStep):
+    """Multivariate elliptical slice sampler step.
+
+    Parameters
+    ----------
+    vars : list
+        List of variables for sampler.
+    prior_cov : array
+        Covariance matrix of the multivariate Gaussian prior.
+    logp : function
+        Log likelihood.
+    model : PyMC Model
+        Optional model for sampling step. Defaults to None (taken from context).
+    """
+    default_blocked = False
```

Comment:
As this operates on the full join, you need to set this to `True`. Otherwise, a separate ESS will be used for each RV.